sound/soc/sof/topology.c

```diff
@@ -2476,6 +2476,10 @@ static const struct snd_soc_tplg_ops sof_dspless_tplg_ops = {
 };
 
 static const struct card_tplg_ops card_ops[] = {
+	{
+		.card_name = "sof-soundwire",
```

Comment:
`sof-soundwire` is kind of reserved as "INTEL"-sof-soundwire, right? Can we add comment to spell this out?